src/main/resources/tags/block_placer_ignored_materials.json

```
@@ -2,16 +2,60 @@
 	"values" : [
 		"#slimefun:sensitive_materials",
 		"#slimefun:mushrooms",
+		"#minecraft:tall_flowers",
 		"#minecraft:beds",
 		"#minecraft:doors",
-		"minecraft:sugar_cane",
 		"minecraft:bamboo",
-		"minecraft:cactus",
-		"minecraft:lily_pad",
-		"minecraft:dead_bush",
+		"#minecraft:rails",
+		"#minecraft:small_flowers",
+		"#minecraft:coral_blocks",
+		"#minecraft:coral_plants",
+		"#minecraft:small_flowers",
+		"#minecraft:buttons",
+		"#minecraft:banners",
+		"#minecraft:signs",
+		"#minecraft:standing_signs",
+		"#minecraft:wall_signs",
```

This is redundant, signs is literally defined as "standing_signs" + "wall_signs"

```suggestion
		"#minecraft:signs",
```